packages/office-ui-fabric-react/src/components/Text/docs/TextDonts.md

```diff
@@ -0,0 +1,2 @@
+- Don't nest Text components if you want to inherit font styling.
+Use a div or span instead that inherits by default.
```

Comment:
Just a note, these *Donts.md files transform bullets into `<li>` elements. Line 2 probably won't look right on its own line here and should probably be incorporated into line 1.